fdbserver/workloads/ConsistencyCheck.actor.cpp

```diff
@@ -1160,14 +1164,6 @@ struct ConsistencyCheckWorkload : TestWorkload {
 		}
 
 		if (foundExtraDataStore) {
-			// Let the cluster fully recover after rebooting/killing storage servers with extra stores.
-			//
-			// This requires an end-to-end comitting transaction to ensure recovery has started in case
-			// any stateless processes, like the commit proxy, were killed.
-			wait(::success(doEmptyCommit(cx)));
```

Comment:
Thanks for getting rid of this hack. You can also remove the doEmptyCommit function above, too. I only added it for this one call.